workloads/fio.yml

```diff
@@ -0,0 +1,140 @@
+---
+#
+# Runs PVC Scale test on an existing RHCOS cluster
+#
+
+- name: Runs PVC scale on a RHCOS cluster
+  hosts: orchestration
+  gather_facts: true
+  remote_user: "{{orchestration_user}}"
+  vars_files:
+    - vars/fio.yml
+  vars:
+    workload_job: "storage-fio"
```

Comment:
Kind of a nitpick but i would recommend to change this just to `fio`.  So everything fio wise (playbook, vars, name of workload job all match and are not either fio or storage-fio.)

Reply:
will track it here, https://github.com/openshift-scale/workloads/issues/78 - do not have cluster to verify change.